pact/Pact/Core/IR/Term.hs

```diff
@@ -561,6 +577,18 @@ traverseModuleTerm f m =
   (mDefs . traversed) (traverseDefTerm f) m
 
 
+traverseIfDefTerm
```

Comment:
Are all the cases of this traversal correct? Or do `IfDfun` and `IfDPact` potentially hold builtins? If those variants can contain builtins, does the traversal need to traverse into them, in the same way it traverses into `IfDConst`?

I could be mixed up about interfaces (maybe they never contain code and therefore don't need to be traversed?)

Reply:
Only `IfDConst` constains terms.

Reply:
@imalsogreg Please feel free to re-ask :)